### PR TITLE
Update env-template file

### PR DESCRIPTION
**Pull request by pnowosie**
Follow up to #1069
Fixes PAY-206

### CODE REVIEW

File: .env-template [hunk 1]
```diff
@@ -36,26 +36,28 @@ YAGNA_DATADIR="."
...
+## GlmSync driver
 #GLMSYNC_RINKEBY_RPC_ADDRESS=http://rinkeby-api.zksync.imapp.pl/jsrpc
-#GLMSYNC_FAUCET_ADDR=http://3.249.139.167:5778/zk/donatex
+#GLMSYNC_MAINNET_RPC_ADDRESS=https://api.zksync.imapp.pl/jsrpc
+#GLMSYNC_FAUCET_ADDR=
 
-## Payment accounts
-#ACCOUNT_LIST=accounts.json
+# Ethereum chain: rinkeby or mainnet
+#ETH_FAUCET_ADDRESS=http://faucet.testnet.golem.network:4000/donate
```
Comment (Wiezzel) [inline .env-template:60]: These two lines need to go away.

**Review by Wiezzel — CHANGES_REQUESTED** [review]

**Comment by Wiezzel** [comment]
@pnowosie Sorry, I've mislead you with my solution. `ACCOUNT_LIST` env variable should be included.

**Comment by pnowosie** [comment]
> @pnowosie Sorry, I've mislead you with my suggestion. `ACCOUNT_LIST` env variable should be included.

Correct, [it is still there](https://github.com/golemfactory/yagna/blob/bba8b9393431e9287e68bd2e65ae48be8f4b983a/.env-template#L37).  I've just removed the duplicate

**Comment by Wiezzel** [comment]
@pnowosie Ah, so I've unintentionally posted the right suggestion. That's called *subconscious programming*.

**Review by Wiezzel — APPROVED** [review]

**Review by maaktweluit — APPROVED** [review]
LGTM!

**Pull request merged**